### PR TITLE
[WEB-1026] - update lang switch dropdown

### PR DESCRIPTION
**Pull request by davidejones**
### What does this PR do?

This PR stops relying on .translations for the english menu entry in the lang switcher.
Instead lets always show it when on a non english site as english will always be there.

### Motivation

https://datadoghq.atlassian.net/browse/WEB-1026

### Preview

check footer nav select link in multiple pages in multiple languages
given example was
https://docs-staging.datadoghq.com/david.jones/footer-langlinks/ja/tracing/setup/nginx/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by kayayarai — APPROVED** [review]
Fine from a docs perspective. Awaiting CorpWeb approval before merging

**Review by nsollecito — APPROVED** [review]

**Pull request merged**
